src/api/firebase.js

```diff
@@ -183,12 +184,22 @@ export async function addItem(listPath, { itemName, daysUntilNextPurchase }) {
 	});
 }
 
-export async function updateItem() {
-	/**
-	 * TODO: Fill this out so that it uses the correct Firestore function
-	 * to update an existing item. You'll need to figure out what arguments
-	 * this function must accept!
-	 */
+// User can update items in their list by checking the checkbox associated with an item, and the totalPurchase field increments by 1 in Firebase.
+// The checkbox unchecks 24 hours after the box is checked.
```

Comment:
<!-- Nitpicks are trivial preference-based requests.  -->
**nitpick ( non-blocking ):** Technically, this second sentence isn't relevant to this piece of code.